### PR TITLE
Additions and edits for the Programing Mission

### PR DESCRIPTION
**Pull request by alohaas**
I'm working through the documentation, page by page, and adding descriptive copy about the programming mission as well as links to the new forum where appropriate.

cc: @moollaza @tagawa @edgesince84 @zekiel

### CODE REVIEW

File: ddh-intro.md [hunk 1]
```diff
@@ -37,7 +39,7 @@ We welcome new contributors to dive in and improve live Instant Answers. It's a
 - Tackle some of the "low-hanging fruit" and, "high priority" issues on our [Live Issues Page](https://duck.co/ia/dev/issues?tag=lowhangingfruit)
 - Help other DuckDuckHackers with their [Instant Answers in development](https://duck.co/ia/dev/pipeline)
 - Suggest improvements & file bugs for [live Instant Answers](https://duck.co/ia) through the, "Create Issue" button on each IA page
-
+- Join the discussion in our [forum](https://forum.duckduckhack.com/tags/c/programming/needs-input) about how we can improve Instant Answers for programmers.
```
Comment (zekiel) [inline ddh-intro.md:42]: If this section is for learning, then maybe pointing to specific examples of good brainstorming like the MDN or JS threads in the forum.

File: ddh-intro.md [hunk 2]
```diff
@@ -7,16 +7,18 @@
 [Instant Answers](https://duck.co/ia) are created and maintained by a community of open source developers around the world so that millions of searches on DuckDuckGo (like [who's in space?](https://duckduckgo.com/?q=who%27s+in+space%3F&ia=answer) and [tennis cheatsheet](https://duckduckgo.com/?q=tennis+cheatsheet&ia=cheatsheet&iax=1)) show a perfect result, above any ads or regular links. Just like the Wikipedia community creates articles for topics -- the DuckDuckHack community creates Instant Answers for searches!
 
 1. Use these docs and our help to improve search results for the topics you care about.
-2. Join our [dev list](https://www.listbox.com/subscribe/?list_id=197814) to keep in touch with all the latest news and DuckDuckHack events.
-3. Become a leader and all-star contributor helping to change the world of search through community!
+2. Sign up for our [dev list](https://www.listbox.com/subscribe/?list_id=197814) to keep in touch with all the latest news and DuckDuckHack events.
```
Comment (tagawa) [inline ddh-intro.md]: Could we say something like `to get the latest DuckDuckHack news, contribution suggestions and programming tips` here?

**Comment by moollaza** [comment]
Content aside, the changes LGTM :+1:

**Comment by zekiel** [comment]
lgtm -- pulling a Friday merge :) Please create an issue or submit a pull request for any odd ends.

**Pull request merged**